src/Controller/InsideElifeController.php

```diff
@@ -81,6 +84,17 @@ public function articleAction(Request $request, string $id) : Response
         $arguments['contentHeader'] = $arguments['article']
             ->then($this->willConvertTo(ContentHeader::class));
 
+        $arguments['contextualData'] = $arguments['article']
+            ->then(function (BlogArticle $article) {
+                if (!$this->get('security.authorization_checker')->isGranted('FEATURE_CAN_USE_HYPOTHESIS')) {
+                    return null;
+                }
+
+                $metrics = [new ContextualDataMetric('Comments', 0, 'hypothesis-comment-count')];
```

Comment:
0 is like the default? The non-js version?

Reply:
Yeah, this will be replaced with fetching it from the API server-side.